packages/client/src/StreamIDBuilder.ts

```diff
@@ -37,6 +37,15 @@ function parseRawDefinition(definition: StreamDefinition): [string, number | und
     }
 }
 
+export const streamDefinitionToString = (definition: StreamDefinition) => {
```

Comment:
return type

Reply:
Method name could reveal that it is intended to produce just a description for e.g. logging purposes

Reply:
Fixed